src/GameController/GCMotion.cs

```diff
@@ -0,0 +1,48 @@
+//
+// GCMotion.cs: extensions to GCMotion iOS API
+//
+// Authors:
+//   TJ Lambert (t-anlamb@microsoft.com)
+//
+// Copyright 2019 Microsoft Corporation.
+
+using System;
+
+using ObjCRuntime;
+using Foundation;
+
+namespace GameController {
+
+	// Removed the Headers for these structs because I could not compile with them.
+	// [iOS (13,0), Mac (10,15, onlyOn64: true), TV (13,0)]
+	public struct GCAcceleration {
+
```

Comment:
Not a big deal, but I think this looks better without a newline between each field.

Reply:
Changed!